SCAutolib/models/gui.py

```diff
@@ -0,0 +1,296 @@
+from SCAutolib import run
```

Comment:
Sort imports. Built-in tools in Pycharm or VSCode should be able to do this for you